### DISCUSSION TITLE
Speed lower when Repeater on

### DISCUSSION THREAD

**Discussion by livinlicious** [Mygod/VPNHotspot#445] [Q&A]
Hi God,
I have the app running on a device already, everything works flawless. (Android 13, Poco F3)

Now I have it also installed on an older Xiaomi A2 Lite, which runs on Android 10.

Installed NordVPN, tested speed, result:full speed.

As soon as I turn on repeater, the speed on the device drops to about 1/5th.
The speed on devices connect to it is also that same speed.

I thought maybe the WIFI chip in the A2 is not as good, so I tried using the Wifi on 5ghz and the repeater on 2.4ghz (just using different modes might help?), but that didnt solve the issue.

The phone seems clearly good enough for the VPN decoding, but I think there is some problem with WIFI receiving + sending, that makes this an issue.

Dear GOD - Do you have a suggestion? :)

**Comment by Mygod** [comment]
It might depend on a number of factors. Performance is not a priority for me (for this app) but if you find anything, PRs are welcome.
